tests/main/install-hook-can-run-svcs/task.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Check that install hooks in snaps can start services
+
+execute: |
+  # shellcheck source=tests/lib/snaps.sh
+  . "$TESTSLIB"/snaps.sh
+
+  echo "Verify that the snap installs"
+  install_local test-snapd-install-hook-runs-svc
+
+  echo "Verify that the snap service is still disabled"
+  snap services | MATCH "test-snapd-install-hook-runs-svc\\.svc\\s+disabled\\s+inactive"
+
+  echo "Verify that the snap service ran and did something during the install hook"
+  MATCH snap-svc-did-this < /var/snap/test-snapd-install-hook-runs-svc/current/data
```

Comment:
Could you please rename this test to _services-install-hook-can..._ ? It's quite useful to have all services-related tests grouped like this (I renamed them once).

Reply:
sure